samples/Kmd.Logic.DocumentGeneration.Client.ConfigurationSample/appsettings.json

```diff
@@ -6,7 +6,7 @@
   },
   "DocumentGeneration": {
     "SubscriptionId": "00000000-0000-0000-0000-000000000000",
-    "DocumentGenerationServiceUri": "https://gateway.kmdlogic.io/document-generation/v2"
```

Comment:
This is not the same v2.
This one is the version of the API in APIM.

Reply:
Fixed in 01522a4eaf410099e003bcc977431c76fee743e4